helm/values.yaml

```diff
@@ -1,7 +1,7 @@
 # configure connaisseur deployment
 deployment:
   replicasCount: 3
-  image: securesystemsengineering/connaisseur:v1.4.5
+  image: securesystemsengineering/connaisseur:v1.5.0
```

Comment:
From the single entry to list change in the config, I would assume this breaks existing configs and would assume 2.0.0 or does `self.notaries = [Notary(notary) for notary in config_content]` parse both a single notary and a notary that is a list?

Reply:
Nope, especially if we change the `notary` field into `notaries`. I'll go with version 2.0.0